metis-common/src/main/java/eu/europeana/metis/utils/ExternalRequestUtil.java

```diff
@@ -29,8 +29,8 @@
 public final class ExternalRequestUtil {
```

Comment:
I left them for now, since I'm not sure where they might be used externally from our applications, I know that they were used somewhere else but I don't remember exactly where.
Also we might want to use them, so I would keep them for a while

Reply:
OK. Perhaps they are used in the sandbox or in Clio?

Reply:
Last I know they were used somewhere from Helin